apps/deck-playground/layers/custom-reference/CustomReferenceLayer.ts

```diff
@@ -0,0 +1,19 @@
+import { EditableGeoJsonLayer } from '@nebula.gl/layers'
+import { DrawPolygonMode, ModifyMode } from '@nebula.gl/edit-modes'
+
+export class CustomReferenceLayer extends EditableGeoJsonLayer {
+  static layerName = 'custom-reference-layer'
+  renderLayers() {
+    const { mode, data, selectedFeatureIndexes, onEdit, onClick } = this.props
+    return [
+      new EditableGeoJsonLayer({
+        id: 'custom-reference',
+        data,
+        onEdit,
+        onClick,
+        selectedFeatureIndexes,
+        mode: mode === 'edit' ? ModifyMode : DrawPolygonMode,
```

Comment:
I got this error `Deprecated use of passing `mode` as a string. Pass the mode's class constructor instead.` but as I see here you are not using a string. any idea what is going on?

Reply:
Looking into this 👁️ 🐽 👁️